src/script_pokemon_util.c

```diff
@@ -325,12 +320,9 @@ void ToggleGigantamaxFactor(struct ScriptContext *ctx)
     {
         bool32 gigantamaxFactor;
 
-        species = GetMonData(&gPlayerParty[partyIndex], MON_DATA_SPECIES);
-        for (i = 0; i < ARRAY_COUNT(sGigantaxFactorLockedSpecies); i++)
-        {
-            if (species == sGigantaxFactorLockedSpecies[i])
-                return;
-        }
+        species = SanitizeSpeciesId(GetMonData(&gPlayerParty[partyIndex], MON_DATA_SPECIES));
+        if (gSpeciesInfo[species].isMythical)
```

Comment:
Do we really need species as separate variable here if it's only used once?

Reply:
Nope, removed.